src/smc-webapp/entry-point.coffee

```diff
@@ -28,11 +26,12 @@ require('./landing-actions')
 # Makes some things work. Like the save button
 require('./jquery_plugins')
 
+###
 # Initialize app stores, actions, etc.
+###
 require('./init_app')
-
-# Initialize the account store.
-require('./account')
+require('./account').init(redux)
+require('./webapp-hooks.ts')
```

Comment:
Why is there an explicit ".ts" there?  I'm pretty sure that is a warning or error (depending on what is compiling this)...